compiler-rt/test/asan/TestCases/fakestack_alignment.cpp

```diff
@@ -0,0 +1,53 @@
+// RUN: %clangxx_asan -fsanitize-address-use-after-return=always -O0 %s -o %t && %run %t 2>&1
+// XFAIL: *
+
+#include <assert.h>
+#include <pthread.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+struct alignas(4096) page {
```

Comment:
this is unused? Can we merge this with the other test, and make the alignas / thread count a define instead?